scripts/feature_tests/upgrade/upgrade_common.sh

```diff
@@ -148,10 +144,12 @@ EOF
 }
 
 function manage_node_taints() {
+    kubectl get secrets "${CLUSTER_NAME}"-kubeconfig -n "${NAMESPACE}" -o json | \
+    jq -r '.data.value'| base64 -d > /tmp/kubeconfig-"${CLUSTER_NAME}".yaml
```

Comment:
```suggestion
        jq -r '.data.value'| base64 -d > /tmp/kubeconfig-"${CLUSTER_NAME}".yaml
```

Reply:
shall be edited. Not before merge because the CI is so difficult

Reply:
fixed in https://github.com/metal3-io/metal3-dev-env/pull/497